db/migrate/20160324124129_add_index.rb

```diff
@@ -0,0 +1,4 @@
+class AddIndex < ActiveRecord::Migration
+  def change
```

Comment:
do you want to take this over because clearly i dont know what i am doing. I was suspect as to why there was nothing generated in AddIndex.. Not sure why it wouldn't let me create another Subject of the same name though..